plugins/src/1.5/scala/com/stratio/sparta/plugin/output/mongodb/MongoDbOutput.scala

```diff
@@ -62,7 +59,7 @@ class MongoDbOutput(keyName: String,
     override def save(dataFrame: DataFrame, saveMode: SaveModeEnum.Value, options: Map[String, String]): Unit = {
       val tableName = getTableNameFromOptions(options)
       val timeDimension = getTimeFromOptions(options)
-      val dataFrameOptions = getDataFrameOptions(tableName, dataFrame.schema, timeDimension, saveMode))
+      val dataFrameOptions = getDataFrameOptions(tableName, dataFrame.schema, timeDimension, saveMode)
```

Comment:
It will be nice to compile all src plugins regardless of the version , to avoid issues like this.